artifactory/commands/transferfiles/utils.go

```diff
@@ -326,6 +333,22 @@ func updateThreads(pcWrapper *producerConsumerWrapper, buildInfoRepo bool) error
 	return nil
 }
 
+// Interrupt the transfer by populating the stopSignal channel with the Interrupt signal if '~/.jfrog/transfer/stop' file exist.
```

Comment:
...if '~/.jfrog/transfer/stop' file exist. --> ...if the '~/.jfrog/transfer/stop' file exists.